### PR TITLE
New package: typecatcher-0.4

### PR DESCRIPTION
**Pull request by hervyqa**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86-64)

Screenshot_20211116_112823

### CODE REVIEW

File: srcpkgs/typecatcher/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'typecatcher'
+pkgname=typecatcher
+version=0.4
+revision=1
+build_style=python3-module
+hostmakedepends="intltool python3-distutils-extra python3-gobject gobject-introspection"
+makedepends="gtk+3-devel webkit2gtk-devel "
+depends="gtk+3 python3 python3-gobject webkit2gtk yelp"
```
Comment (abenson) [inline srcpkgs/typecatcher/template:8]: You can drop `gtk+3` and `webkit2gtk` here, they'll be autodetected.

File: srcpkgs/typecatcher/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+short_desc="Download Google webfonts for off-line use"
+maintainer="Hervy Qurrotul Ainur Rozi <hervyqa@pm.me>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/andrewsomething/typecatcher"
+distfiles="https://github.com/andrewsomething/${pkgname}/archive/refs/tags/${version}.tar.gz"
```
Comment (abenson) [inline srcpkgs/typecatcher/template:13]: Don't use `${pkgname}` here.

File: srcpkgs/typecatcher/template (hunk 1 above)
Comment (hervyqa) [reply srcpkgs/typecatcher/template:8]: @abenson ok wait..

File: srcpkgs/typecatcher/template (hunk 2 above)
Comment (hervyqa) [reply srcpkgs/typecatcher/template:13]: @abenson thanks.. 👍

**Comment by hervyqa** [comment]
ok. done.. 🎉

**Pull request closed without merging**